templates/common/bin/common.sh

```diff
@@ -65,8 +65,8 @@ function common_ironic_config {
     # Can we just put custom.conf in something like /etc/ironic/ironic.conf.d/custom.conf
     # and have it automatically detected, or would we have to somehow change the call
     # to the ironic binary to tell it to use that custom conf dir?
-    echo merging /var/lib/config-data/default/custom.conf into ${SVC_CFG_MERGED}
-    crudini --merge ${SVC_CFG_MERGED} < /var/lib/config-data/default/custom.conf
+    echo merging /var/lib/config-data/merged/01-ironic-custom.conf into ${SVC_CFG_MERGED}
```

Comment:
I think we can drop this. We are merging with crudini into `/var/lib/config-data/merged/ironic.conf` which is later mapped to `/etc/ironic/ironic.conf` - But we are also monting 01-ironic-custom.conf into `"/etc/ironic/ironic.conf.d/01-ironic-custom.conf"` effectively that would have the same result.